pkg/health/ping.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 The Service Manager Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package health
+
+// pingIndicator is a default indicator that always returns up
+type pingIndicator struct {
+}
+
+func (*pingIndicator) Name() string {
+	return "ping"
```

Comment:
PingHealthIndicator (if these will be part of logs, more complete names will be more useful)